deploy/operator.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       containers:
         - name: nxiq-operator-certified
           # Replace this with the built image name
-          image: registry.connect.redhat.com/sonatype/nxiq-operator-certified:1.94.0-4
+          image: quay.io/jflinchbaugh/nxiq-operator-certified:1.95.0-1
```

Comment:
leftovers from testing?

Reply:
yep. i also caught this while i was reviewing and annotating the PR.